orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -61,9 +61,12 @@ def __init__(self, module):
         self._save_onnx_prefix = ''
 
         # Graph transformer config
+        # Specify cast propagation strategy. Currently two strategies are available, insert-and-reduce and flood-fill
+        # the default is insert-and-reduce
+        self._propagate_cast_ops_strategy = C.PropagateCastOpsStrategy.INSERT_AND_REDUCE
         # Optimize by moving Cast operations if propagate_cast_ops_level is non-negative.
-        # Use predetermined list of opcodes considered safe to move before/after cast operation
-        # if propagate_cast_ops_level is positive and use propagate_cast_ops_allow otherwise.
+        # In addition to propagate_cast_ops_allow use predetermined list of opcodes considered safe to move before/after cast operation
+        # if propagate_cast_ops_level is positive and use only propagate_cast_ops_allow otherwise.
```

Comment:
This comment is confusing

Reply:
+1
and also what's the relationship between the two settings and the level setting below